### PR TITLE
Add slack invite link

### PR DESCRIPTION
**Pull request by swapagarwal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ A curated list of awesome Go frameworks, libraries and software. Inspired by [aw
 
 Please take a quick gander at the [contribution guidelines](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md) first. Thanks to all [contributors](https://github.com/avelino/awesome-go/graphs/contributors); you rock!
 
-[Join us on Slack](https://gophers.slack.com/messages/awesome/) to chat with other awesome-go maintainers!
+[Join us on Slack](https://gophers.slack.com/messages/awesome/) to chat with other awesome-go maintainers! ([Request an invite](http://bit.ly/go-slack-signup), then join the #awesome channel.)
```
Comment (shurcooL) [inline README.md]: Perhaps using https would be better.

Comment (swapagarwal) [inline README.md]: Done!

**Comment by shurcooL** [comment]
LGTM.

**Comment by avelino** [comment]
who will be responsible in sending invitation?

**Comment by dgryski** [comment]
The invite is automatically sent by the form at the link.

**Pull request merged**